packages/react/src/components/ComposedModal/next/ComposedModal.js

```diff
@@ -6,10 +6,76 @@ import { ModalFooter } from '../ComposedModal';
 import classNames from 'classnames';
 
 import toggleClass from '../../../tools/toggleClass';
+import requiredIfGivenPropIsTruthy from '../../../prop-types/requiredIfGivenPropIsTruthy';
 
 import wrapFocus from '../../../internal/wrapFocus';
 import { usePrefix } from '../../../internal/usePrefix';
 
+export function ModalBody(props) {
+  const {
+    className,
```

Comment:
With function declarations, the style would be to restructure props in the argument position like:

```jsx
function ModalBody({ className, ...rest }) {
```